pandas/core/window/rolling.py

```diff
@@ -544,7 +569,6 @@ def _apply(
                     continue
                 else:
                     raise DataError("No numeric types to aggregate") from err
-
```

Comment:
Could you undo this whitespacing?

Reply:
Should be an autoformat I think. Probably autoformat is missing or is missing an idempotent test.

> @cottrell sorry this hasn't seen any attention; we have a number of PRs.
> 
> conceptually this would be ok. I would want to refactor out the actual quantile code to make the multiple loops in cython reasonable.
> 
> Does this change the perf of the single quantile case?
> 
> the return value here is also slightly tricky as now we'd have to return a frame (rather than a series), but that's prob ok (I think we do that anyhow for regular quantile).
> 
> please merge master.

Also should this branch not have been rebased? I thought that was the pattern pandas dev was following?